packages/build/src/plugins_core/edge_functions/index.js

```diff
@@ -69,6 +65,12 @@ const coreStep = async function ({
 
   await validateEdgeFunctionsManifest({ buildDir, constants: { EDGE_FUNCTIONS_DIST: distDirectory } })
 
+  if (!isRunningLocally) {
```

Comment:
We should be checking that we actually have edge functions, because `declarations` can be an empty array.